dashboard/src/app/index.module.ts

```diff
@@ -84,7 +84,7 @@ function keycloakLoad(keycloakSettings: any) {
       return cheBranding.ready.then(() => {
         reject(`<div  class="error-header"><span>Certificate Error</span><a href="/"><i class="fa fa-times"></i></a></div>
  <div class="error-body"><p>Your Che host may be signed with a self-signed certificate. To resolve this issue, try these possible solutions:</p>
- <p>1.) Import CA certificate info into your browser. You can find instructions on how to do this in you 
+ <p>1.) Import CA certificate info into your browser. You can find instructions on how to do this in you
```

Comment:
```suggestion
 <p>1.) Import CA certificate info into your browser. You can find instructions on how to do this in
```
?

Reply:
@benoitf Thanks for suggestion! I won't accept it because whole this error message is going to be changed in [this PR](https://github.com/eclipse/che/pull/16293) ([diff](https://github.com/eclipse/che/pull/16293/files#diff-23910a1448372bee6229c74377877dfdR85-R90))